app/views/groups/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<header>
+  <%= link_to groups_path, class: 'back-btn' do %>
+    <ion-icon name="arrow-back" class="settings-btn"></ion-icon>
+  <% end %>
+  <h1 class="page-title">New Category</h1>
+  <%= button_to "Sign Out", destroy_user_session_path, method: :delete, class: "sign-out" %>
+</header>
+
+<%= form_with model: @new_group, data: { turbo: false } do |f| %>
+<section class="fields">
+  <div class="field">
+    <%= f.text_field :name, placeholder: "Name" %>
+  </div>
+
+  <div class="field">
+    <%= f.text_field :icon, placeholder: "Icon URL" %>
+  </div>
+  
+  <%= f.hidden_field :user_id, value: current_user.id %>
+  
+  <%= f.submit "CREATE", class: "btton" %>
+  </section>
```

Comment:
- [ ] It would be nice if you could improve the UI of this page so elements are well displayed on the screen as required. Alignment and spacing of elements are key to a good user experience of your app. Please reduce the font size of the heading and add a little padding on the signout button so it appears as a button instead of a badge 👍🏽
- [ ] Please also add some padding on the input fields for creating a category so there is some breathing space in the design. It will improve the look and feel 👍🏽

<img width="486" alt="Screenshot 2023-07-15 at 6 46 32 PM" src="https://github.com/brainskev/Budget_App/assets/50941074/cb9e155f-7518-4ef9-9b9a-cc549e347f90">

Reply:
Thanks for the detailed review will make sure I implement all the requested changes.